attr/validators.py

```diff
@@ -85,3 +85,33 @@ def provides(interface):
     the value it got.
     """
     return _ProvidesValidator(interface)
+
+
+@attributes(repr=False)
+class _OptionalValidator(object):
+    """
+    A validator that makes an attribute optional.  An optional field is one
+    which can be set to None in addition to its intended data type.
```

Comment:
I’m saying you’re selling your validator short. :)

`to None in addition to its intended data type.` is too limiting.  `attrs` may ship only with datatype related validators, but as ppl have plenty of custom ones.  The current doc string makes it sound like optional will not work with them.